src/controller/articlesController.js

```diff
@@ -305,6 +297,18 @@ class ArticleController {
           article.viewStats,
           article.title
         );
+        const getTags = await ArticleTag.findAll({
```

Comment:
This should be `tags` and not `getTags`

Reply:
Thanks for your feedback. It has been changed.